Core/src/main/java/com/plotsquared/core/permissions/Permission.java

```diff
@@ -37,8 +37,6 @@
     PERMISSION_STAR("*"),
     PERMISSION_ADMIN("plots.admin"),
     PERMISSION_ADMIN_AREA_SUDO("plots.admin.area.sudo"),
-    PERMISSION_PROJECTILE_UNOWNED("plots.projectile.unowned"),
-    PERMISSION_PROJECTILE_OTHER("plots.projectile.other"),
```

Comment:
Those should only marked as deprecated for removal.

Reply:
While deprecated should they essentially be an alias for the admin versions or should they not be linked up to anything at all (and only exist in `Permission` I guess)?

Reply:
Leaving it there unused would be the best I think